### PR TITLE
Create images as byte[]

### PR DESCRIPTION
**Pull request by tarekoraby**

### CODE REVIEW

**Comment by Artur-** [comment]
Can this be done in a backwards compatible way?

**Comment by tarekoraby** [comment]
Keep the old classes, and create new `public class BookImageGeneratorByte extends DataType<byte[]>` (and same for food product image)?

**Comment by Artur-** [comment]
I guess so, and reuse the same implementation in both

**Comment by tarekoraby** [comment]
How about now?

**Comment by tarekoraby** [comment]
Should the version be changed in `pom.xml` from `<version>4.1-SNAPSHOT</version>`?

**Comment by Artur-** [comment]
https://github.com/vaadin/example-data/blob/a26a802cd05394290cd4dd1d96a186b956cefc71/src/main/resources/META-INF/resources/frontend/data-generator.ts#L1906 creates BookImageUrl in TS. There should probably be one for `BookImageUrlBytes` also

File: data-generator.tpl.ts [hunk 1]
```diff
@@ -252,6 +257,21 @@ export const DataGenerators: { [key in string]: ValueCreator } = {
       );
     },
   },
+  [DataType.BookImageUrlBytes]: {
+    createValue: (seed, refTime) => {
```
Comment (Artur-) [inline data-generator.tpl.ts]: Can this be a method instead of duplicated?

Comment (tarekoraby) [reply data-generator.tpl.ts]: I tried that, but the whole thing produced some cryptic errors when used in Start. I can try again.

Comment (tarekoraby) [reply data-generator.tpl.ts]: Done

**Comment by Artur-** [comment]
Simplified a bit

**Pull request merged**